cmd/probe/probe.go

```diff
@@ -28,6 +28,7 @@ const (
 
 // RegisteredProbes contains register function of probes which we want to register
 var RegisteredProbes = []func(){
```

Comment:
for better code readability please reorder this list in the priority order (next PR).